### ISSUE TITLE
question: Potential false positive? aws-rds-encrypt-cluster-storage-data when storage_encrypted=true, but no kms key

### ISSUE THREAD

**Issue by PB-TW** [aquasecurity/tfsec#1491]
When an `aws_rds_cluster` has `engine_mode` set to `serverless`, its `storage_encrypted` defaults to `true`. However, since 1.0.0, I get https://aquasecurity.github.io/tfsec/v1.0.11/checks/aws/rds/encrypt-cluster-storage-data/ when running tfsec and no `kms_key_id` is set; even when `storage_encrypted` is explicitly set to true. **Is it a must-have to specify a custom kms key for database encryption?**

If yes, maybe rephrase the problem to use a custom kms key. Since this sentence `There is no encryption specified or encryption is disabled on the RDS Cluster.` does not hold true for:

```
resource "aws_rds_cluster" "example" {
   name = "bar"
...
   storage_encrypted = true
}
```

**Comment by gabrielleecredera** [comment]
yes, the check performed checks that:
- `storage_encrypted` == true AND
- `kms_key_id` is set

**Comment by liamg** [comment]
I've raised an issue in defsec to update the rule documentation to mention the need to set a customer-managed key, as this is admittedly very unclear at the moment: https://github.com/aquasecurity/defsec/issues/202
